client/landing/gutenboarding/components/domain-picker/suggestion-item.tsx

```diff
@@ -15,6 +15,10 @@ interface Props {
 	onSelect: ( domainSuggestion: DomainSuggestion ) => void;
 }
 
+// foobar.com returns ['foobar.com', 'foobar', '.com'];
+// foobar.wordpress.com returns ['foobar.wordpress.com', 'foobar', '.wordpress.com'];
+const rxDomainParts = /([^.]*)(.*)/;
```

Comment:
Looks like this would work also for domains like `*.design.blog` and `*.co.uk` 👍

Reply:
Simple and elegant. If someone wants to write tests for this, they might use this list https://wordpress.com/support/domains/domain-pricing-and-available-tlds/. Maybe write a comment about this?